### DISCUSSION TITLE
async base [python]

### DISCUSSION THREAD

**Discussion by zawsq** [deta/cloud-docs#303] [Help]
probably a stupid question
anyway to add data without fetching it again

![image](https://user-images.githubusercontent.com/60060180/177105207-ba48bb2c-5d22-4ade-9c92-3c46cd960605.png)
becomes like this:
![image](https://user-images.githubusercontent.com/60060180/177105232-20814927-fd29-4616-8de7-c52d0a051fea.png)

so i dont need to keep fetching and adding/removing data again next time on big database before i use put

**Comment by abdelhai** [comment]
hi sorry for the late response, didn't see the message. could you please elaborate, i'm afraid i don't understand the issue here.

**Comment by jnsougata** [comment]
you can update `(set)` any value to existing key using `update` check SDK docs **[here](https://docs.deta.sh/docs/base/sdk/#update)**
![image](https://user-images.githubusercontent.com/53375272/178035870-55c1dc01-8965-42c5-b0fc-b2fc081be192.png)

btw the example you've shown here is not possible as a JSON object can't hold multiple same keys (`id` in your case).
